### PR TITLE
DCES-537-xsd fix

### PR DESCRIPTION
**Pull request by LAA-Dave-Edwards**
## What

Changing int to Integer for ids and the breathing space amount.
Changing a large number of things to decimal as they're stored as NUMBER(n,2), which byte/int/etc wouldn't handle.

[Link to story](https://dsdmoj.atlassian.net/browse/DCES-537)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [x] Tests should be passing: `./gradlew test`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [x] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [x] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: dces-drc-integration/src/main/resources/schemas/contributionsLine.xsd [hunk 1]
```diff
@@ -279,7 +278,7 @@
                         <xsd:element name="propertyDescriptor">
                             <xsd:complexType>
                                 <xsd:sequence>
-                                    <xsd:element type="xsd:byte" name="bedRoomCount"/>
+                                    <xsd:element type="xsd:string" name="bedRoomCount"/>
```
Comment (LAA-Dave-Edwards) [inline dces-drc-integration/src/main/resources/schemas/contributionsLine.xsd:281]: Can contain things like "6+"

File: dces-drc-integration/src/main/resources/schemas/contributionsLine.xsd [hunk 2]
```diff
@@ -517,7 +516,7 @@
                                     <xsd:element type="xsd:string" name="status"/>
                                     <xsd:element type="xsd:date" name="bsStartDate"/>
                                     <xsd:element type="xsd:date" name="bsEndDate"/>
-                                    <xsd:element type="xsd:integer" name="debtAmt"/>
+                                    <xsd:element type="xsd:long" name="debtAmt"/>
```
Comment (LAA-Dave-Edwards) [inline dces-drc-integration/src/main/resources/schemas/contributionsLine.xsd:519]: DB stores no decimal places, but can hold 10 digits.

**Review by dconneely — APPROVED** [review]
Most unexpected cases explained in comments, and applicant/bankDetails/accountNo is an integer in the DB too, so that's also good!.

File: dces-drc-integration/src/main/resources/schemas/contributionsLine.xsd (hunk 2 above)
Comment (dconneely) [reply dces-drc-integration/src/main/resources/schemas/contributionsLine.xsd:519]: `xsd:integer` is unlimited precision (it's only `xsd:int` that's limited to 32 bits).

**Pull request merged**